<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkerBrowserConfigPage</class>
 <widget class="QFrame" name="MarkerBrowserConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marker Database Browser</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin" stdset="0">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Marker Browser</string>
     </property>
     <layout class="QGridLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Maximum number of markers to show</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>Window      </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbx_window">
        <item>
         <property name="text">
          <string>Don't change</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fit context cell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fit marker with margin ..</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center marker</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Center marker with size ..</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QComboBox" name="cbx_context">
        <item>
         <property name="text">
          <string>Any Cell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker Database Top Cell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Current Cell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Current or Any Cell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marker's Original Cell</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>Context</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" colspan="3">
       <widget class="QLineEdit" name="le_max_markers"/>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="lay::MarginWidget" name="mgn_window" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>lay::MarginWidget</class>
   <extends>QWidget</extends>
   <header>layWidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbx_context</tabstop>
  <tabstop>cbx_window</tabstop>
  <tabstop>le_max_markers</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
